tools/miniasm/miniasm.xml

```diff
@@ -0,0 +1,102 @@
+                <option value="advanced">Yes, see full parameter list.</option>
+            </param>
+            <when value="simple">
+            </when>
+            <when value="advanced">
+            	<param name="minMatch" type="integer" value="100" optional="true" label="description" help=""/>
+				<param name="minIden" type="float" value="0.05" optional="true" label="description" help=""/>
+				<param name="minSpan" type="integer" value="1000" optional="true" label="description" help=""/>
+				<param name="minCov" type="integer" value="3" optional="true" label="description" help=""/>
+				<param name="minOvlp" type="integer" value="1000" optional="true" label="description" help=""/>
+				<param name="maxHang" type="integer" value="1000" optional="true" label="description" help=""/>
+				<param name="intThres" type="float" value="0.08" optional="true" label="description" help=""/>
+				<param name="maxGapDiff" type="integer" value="1000" optional="true" label="description" help=""/>
+				<param name="maxDubDist" type="integer" value="50000" optional="true" label="description" help=""/>
+				<param name="minUtgSize" type="integer" value="4" optional="true" label="description" help=""/>
+				<param name="nRounds" type="integer" value="3" optional="true" label="description" help=""/>
+				<param name="finalDropRatio" type="float" value="0.8" optional="true" label="description" help=""/>
+            </when>
+        </conditional>
+    </inputs>
+    <outputs>
+        <data name="gfa" format="txt" />
+    </outputs>
+    <tests>
+        <test>
+	     <param name="readFile" value="mini_reads.gf"/>
+	     <param name="paf" value="mini_reads.paf.gz"/>
+	     <output name="gfa" file="mini_reads.gfa"/>
+	</test>
+    </tests>
+    <help><![CDATA[
+        
+Usage:   miniasm [options] <mapping.paf>|<output.gfa>
+Options:
+  Pre-selection:
+    -m INT      min match length [100]
+    -i FLOAT    min identity [0.05]
+    -s INT      min span [2000]
+    -c INT      min coverage [3]
+  Overlap:
+    -o INT      min overlap [same as -s]
+    -h INT      max over hang length [1000]
+    -I FLOAT    min end-to-end match ratio [0.8]
+  Layout:
+    -g INT      max gap differences between reads for trans-reduction [1000]
+    -d INT      max distance for bubble popping [50000]
+    -e INT      small unitig threshold [4]
+    -f FILE     read sequences []
+    -n INT      rounds of short overlap removal [3]
+    -r FLOAT[,FLOAT]
+                max and min overlap drop ratio [0.7,0.5]
+    -F FLOAT    aggressive overlap drop ratio in the end [0.8]
+  Miscellaneous:
+    -p STR      output information: bed, paf, sg or ug [ug]
+    -b          both directions of an arc are present in input
+    -1          skip 1-pass read selection
+    -2          skip 2-pass read selection
+    -V          print version number
+
+    ]]></help>
+    <citations>
+        <citation type="bibtex">
+@misc{githubseqtk,
```

Comment:
en kun je een citation toevoegen? :) dit is doi link volgens mij `10.1093/bioinformatics/btw152`